node/src/command.rs

```diff
@@ -173,22 +173,32 @@ macro_rules! construct_async_run {
 				})
 			}
 			Runtime::Testnet => {
-				runner.async_run(|$config| {
-					let $components = new_partial::<pop_runtime_testnet::RuntimeApi>(
-						&$config
-					)?;
-					let task_manager = $components.task_manager;
-					{ $( $code )* }.map(|v| (v, task_manager))
-				})
+				#[cfg(feature = "ismp")]
+				unimplemented!("ISMP is not supported in testnet");
```

Comment:
Perhaps https://internals.rust-lang.org/t/mutually-exclusive-feature-flags/8601/2 might negate the need for this? That is, a compilation error rather than runtime panic.

Probably doesnt make sense tho.